.github/workflows/ci.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
       - run: sudo apt update
         if: matrix.os == 'ubuntu-latest'
+      - run: sudo apt install -y libgbm1
```

Comment:
https://github.com/puppeteer/puppeteer/commit/5c839f5e48dffd57cbf740f4a980b154682a2d87

Reply:
Maybe add a comment so we know where this requirement comes from.

Also are there nightmare dependencies that can be removed now?

Reply:
According to the docs puppeteer also requires xvfb
https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-on-travis-ci

Reply:
Looks like puppeteer doesn't need xvfb https://github.com/FredrikNoren/ungit/pull/1336/commits/3a3af0d6fc1447ec9b150a0d8aeed9d4f6621005

- [GitHub Actions Linux](https://github.com/FredrikNoren/ungit/pull/1336/checks?check_run_id=629647416)
- [Travis Linux](https://travis-ci.org/github/FredrikNoren/ungit/jobs/680990378)